### PR TITLE
make boostrap functions async only

### PR DESCRIPTION
**Pull request by alexandrebodin**
#### Description of what you did:
make boostrap functions async only.

Users will need to update there boostrap.js file

#### My PR is a:

- [x] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: examples/getstarted/config/functions/bootstrap.js [hunk 1]
```diff
@@ -8,6 +8,4 @@
  * run jobs, or perform some special logic.
  */
 
-module.exports = cb => {
-  cb();
-};
+module.exports = () => {};
```
Comment (lauriejim) [inline examples/getstarted/config/functions/bootstrap.js]: should we add `async` here ?

File: packages/strapi-generate-new/lib/resources/files/config/functions/bootstrap.js [hunk 2]
```diff
@@ -8,6 +8,4 @@
  * run jobs, or perform some special logic.
  */
 
-module.exports = cb => {
-  cb();
-};
+module.exports = () => {};
```
Comment (lauriejim) [inline packages/strapi-generate-new/lib/resources/files/config/functions/bootstrap.js]: And same here

File: examples/getstarted/config/functions/bootstrap.js (hunk 1 above)
Comment (alexandrebodin) [reply examples/getstarted/config/functions/bootstrap.js]: It doesn't have to be async the function can just return a promise.

File: packages/strapi-generate-new/lib/resources/files/config/functions/bootstrap.js (hunk 2 above)
Comment (alexandrebodin) [reply packages/strapi-generate-new/lib/resources/files/config/functions/bootstrap.js]: It doesn't have to be async the function can just return a promise.

**Pull request merged**